.github/workflows/python-tests.yml

```
@@ -21,16 +21,43 @@ jobs:
+        mkdir -p data/sessions
+        mkdir -p data/llmops
+        mkdir -p data/temp
+        
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
         python -m pip install pytest pytest-cov
         if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+        if [ -f requirements-dev.txt ]; then pip install -r requirements-dev.txt; fi
         pip install -e .
     
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)